tests/functional/posting_test.php

```diff
@@ -72,6 +72,20 @@ public function test_html_entities()
 		$this->assertContains('&#128512;', $crawler->text());
 	}
 
+	public function test_quote()
```

Comment:
That makes sure that it is working as expected. If you're sure that enough... then fine by me.